### PR TITLE
Create ShEx for Care Pathway and Disease History and Diagnosis

### PR DESCRIPTION
**Pull request by sxzhang1201**
The examples have been validated against the ShEx without conflicts. Thanks for reviewing.

### CODE REVIEW

File: shex/sio-model/diseaseHistoryAndDiagnosisShape.shex [hunk 1]
```diff
@@ -1,27 +1,43 @@
...
+  a [sio:SIO_000498];
+  sio:SIO_000228 @:personRoleShape;
+  sio:SIO_000217 @:ageAtDiagnosisQualityShape;
+  sio:SIO_000217 @:symptomOnsetQualityShape
 }
 
 :personRoleShape IRI {
   a [obo:OBI_0000093];
-  sio:SIO_000356 @:medicalDiagnosisProcessShape
+  sio:SIO_000356 @:processShape +
```
Comment (sxzhang1201) [inline shex/sio-model/diseaseHistoryAndDiagnosisShape.shex:18]: "+" means one or more: one role should have at least one process.

File: shex/sio-model/diseaseHistoryAndDiagnosisShape.shex [hunk 2]
```diff
@@ -1,27 +1,43 @@
...
 }
 
-:medicalDiagnosisProcessShape IRI {
-  a [sio:SIO_001001];
-  dct:date  xsd:dateTime;
-  sio:SIO_000229 @:SQDiagnosisShape
+:processShape IRI {
+  a [sio:SIO_000006 sio:SIO_001001];
+  prov:wasInformedBy @:processShape ?;
```
Comment (sxzhang1201) [inline shex/sio-model/diseaseHistoryAndDiagnosisShape.shex:23]: "?" means zero or one: the "medical diagnosis process" in the figure has no was-informed-by triple, thus allowing the "zero".

File: shex/sio-model/diseaseHistoryAndDiagnosisShape.shex [hunk 3]
```diff
@@ -1,27 +1,43 @@
...
-  sio:SIO_000356 @:medicalDiagnosisProcessShape
+  sio:SIO_000356 @:processShape +
 }
 
-:medicalDiagnosisProcessShape IRI {
-  a [sio:SIO_001001];
-  dct:date  xsd:dateTime;
-  sio:SIO_000229 @:SQDiagnosisShape
+:processShape IRI {
+  a [sio:SIO_000006 sio:SIO_001001];
```
Comment (sxzhang1201) [inline shex/sio-model/diseaseHistoryAndDiagnosisShape.shex:22]: if the  "process_1" in the figure (<https://github.com/ejp-rd-vp/CDE-semantic-model/blob/develop/images/sio-model/5_Disease_history_and_diagnosis.png>) was a type of process (i.e., `sio:SIO_00006`) and had the `rdf:type sio:process` triple, the ShEx would be changed.

File: shex/sio-model/diseaseHistoryAndDiagnosisShape.shex (hunk 1 above)
Comment (rajaram5) [reply shex/sio-model/diseaseHistoryAndDiagnosisShape.shex:18]: Sometime to check with @markwilkinson . In the example turtle I connected multiple process to same role. For now we use this pattern we might change it based on next week's review.

**Review by rajaram5 — APPROVED** [review]
Looks good to me. @sxzhang1201 can you merge it?

**Pull request merged**